.docs/README.md

```diff
@@ -196,45 +181,42 @@ $payment = $service->createPayment([
 ]);
 ```
 
-Zadruhé nastavit adresy, na které Gopay platební brána přesměruje při úspěchu či
-naopak selhání platby.
+Second, set the addresses to which the Gopay payment gateway redirects the success of the payment or while error in payment proccess.
 
 ```php
 $service->setSuccessUrl($this->link('//success', ['orderId' => $orderId]));
 $service->setFailureUrl($this->link('//failure', ['orderId' => $orderId]));
 ```
 
-Je užitečné si poznačit ID platby (například pokud se má platba vázat
-k nějaké objednávce apod.). Toho lze docílit předáním callbacku jako třetího
-parametru metodě `pay()`.
+It is useful to make a note of the payment ID (for example if the payment is to be bound to an order etc.). This can be accomplished by passing the callback as the third parameter to the `pay ()` method.
 
 ```php
 $storeIdCallback = function ($paymentId) use ($order) {
 	$order->setPaymentId($paymentId);
 };
 ```
 
-Samotné placení lze provést dvěma způsoby.
+The payment itself can be made in two ways:
 
-### REDIRECT brána
+### Redirect after a payment
 
 ```php
 $response = $gopay->pay($payment, $gopay::METHOD_TRANSFER, $storeIdCallback);
 ```
 
-Akce `pay()` vrátí `Response` objekt. Resp. `RedirectResponse`, který vás přesměruje na Gopay bránu.
+The `pay ()` action returns an `RedirectResponse`, which redirects you to the Gopay gateway.
 
 ```php
 $this->sendResponse($response);
 ```
 
-### INLINE brána
+### Inline payment
 
 ```php
 $response = $gopay->payInline($payment, $gopay::METHOD_TRANSFER, $storeIdCallback);
 ```
 
-Akce `payInline()` vám vrátí pole s klíči **url** a **signature**.
+The `payInline ()` action returns a field with the **url** and **signature** keys.
```

Comment:
The `payInline()` action returns a field with the **url** and **signature** keys.